xonsh/readline_shell.py

```diff
@@ -121,6 +121,12 @@ def setup_readline():
             # this seems to fail with libedit
             print_exception('xonsh: could not load readline default init file.')
 
+    # handle case sensitive, see Github issue #1342 for details
+    if env.get('CASE_SENSITIVE_COMPLETIONS'):
+        readline.parse_and_bind("set completion-ignore-case off")
+    else:
+        readline.parse_and_bind("set completion-ignore-case on")
```

Comment:
You're right! Should I move these lines to `ReadlineShell.completedefault()` which does fix dynamic setting. If so, is it too heavy to call `readline.parse_and_bind` once for every cmd?